uaa_bot/client.py

```diff
@@ -269,3 +269,48 @@ def deactivate_user(self, user: dict) -> dict:
         )
 
         return response
+
+    def list_users_last_logon(
+        self,
+        days_ago: int = 0,
+        days_range: int = 365,
+        start_of_day: int = None,
+        end_of_day: int = None,
```

Comment:
the api here feels weird - if we need both ways to call it, maybe it'd be better to have a function that uses the absolute time (e.g, `list_users_last_logged_in_abs(start_ts, end_ts)`, and a wrapper function for relative times `list_users_last_logged_in_rel(start_days_ago, window_length)`?

Reply:
Added this with a recent commit